src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,94 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static User userOne = new User();
+    private static User userTwo = new User();
+    private static User userThree = new User();
+    private static User userFour = new User();
+    private static User userFive = new User();
+    private RegistrationServiceImpl registrationService = new RegistrationServiceImpl();
+
+    @BeforeAll
+    static void beforeAll() {
+        userOne.setLogin("Sakura");
+        userOne.setPassword("12345678");
+        userOne.setAge(16);
+
+        userTwo.setLogin("CherryTree");
+        userTwo.setPassword("12345");
+        userTwo.setAge(19);
+
+        userThree.setLogin("Sakura");
+        userThree.setPassword("123456");
+        userThree.setAge(18);
+
+        userFour.setLogin("ILoveCherry");
+        userFour.setPassword("ireallylovecherry");
+        userFour.setAge(21);
+
+        userFive.setLogin(null);
+        userFive.setPassword(null);
+        userFive.setAge(null);
+    }
```

Comment:
* 1 valid user is enough, you just need to clear DB after each test to reuse it
* initialize it with a `new User()` before each test to avoid having common data for several tests.
* set user some 1 corresponding non-valid field in each `notOk` test case
* `Before All` itself should only initialize `registrationService`
* use constants for user data, so that it would be clear what is valid and what is not